<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="TreeView" name="treeView">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>800</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="expandsOnDoubleClick">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>true</bool>
        </attribute>
       </widget>
       <widget class="TreeView" name="tool">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>700</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>967</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="File">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>工程</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_all"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="system_modelling">
    <property name="title">
     <string>系统建模</string>
    </property>
   </widget>
   <widget class="QMenu" name="Process">
    <property name="title">
     <string>InSAR处理</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>导入数据</string>
     </property>
     <property name="icon">
      <iconset resource="../QtWidgetsApplication3.qrc">
       <normaloff>:/QtWidgetsApplication3/bin/icon/import.png</normaloff>:/QtWidgetsApplication3/bin/icon/import.png</iconset>
     </property>
     <addaction name="actionRepeatPass"/>
     <addaction name="actionSingleTransDoubleRecv"/>
     <addaction name="actionPingPong"/>
     <addaction name="actionDualFreqPingPong"/>
    </widget>
    <widget class="QMenu" name="lvbo">
     <property name="title">
      <string>滤波</string>
     </property>
     <addaction name="actionlvbo1"/>
     <addaction name="actionlvbo2"/>
    </widget>
    <widget class="QMenu" name="jiechan">
     <property name="title">
      <string>解缠</string>
     </property>
     <addaction name="actionjiechan1"/>
     <addaction name="actionjiechan2"/>
    </widget>
    <addaction name="menuImport"/>
    <addaction name="actionRegistration"/>
    <addaction name="actionCut"/>
    <addaction name="actionqucankao"/>
    <addaction name="lvbo"/>
    <addaction name="jiechan"/>
    <addaction name="actiongaocheng"/>
    <addaction name="actiongeocode"/>
   </widget>
   <widget class="QMenu" name="performance_evaluation">
    <property name="title">
     <string>处理性能评估</string>
    </property>
    <addaction name="actioncoregis_evaluate"/>
    <addaction name="actionunwrap_evaluate"/>
    <addaction name="actiondem_evaluate"/>
   </widget>
   <widget class="QMenu" name="menuSAR">
    <property name="title">
     <string>SAR</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuInSAR">
    <property name="title">
     <string>InSAR</string>
    </property>
    <addaction name="actionBaseline_Preview"/>
    <addaction name="actionInterferometric_Formation"/>
    <addaction name="actionDenoise"/>
    <addaction name="actionUnwrap"/>
    <addaction name="actionDEM"/>
   </widget>
   <widget class="QMenu" name="menuDInSAR">
    <property name="title">
     <string>DInSAR</string>
    </property>
    <widget class="QMenu" name="menuSBAS">
     <property name="title">
      <string>SBAS</string>
     </property>
     <property name="icon">
      <iconset resource="../QtWidgetsApplication3.qrc">
       <normaloff>:/QtWidgetsApplication3/bin/icon/baseline_formation.png</normaloff>:/QtWidgetsApplication3/bin/icon/baseline_formation.png</iconset>
     </property>
     <addaction name="actionBaseline_Formation"/>
     <addaction name="actionSBAS_deformation"/>
     <addaction name="actionreference_re_selection"/>
     <addaction name="actionDeformation_Preview"/>
     <addaction name="actionExport_KML"/>
    </widget>
    <addaction name="actionSLC_deramp"/>
    <addaction name="menuSBAS"/>
   </widget>
   <addaction name="File"/>
   <addaction name="system_modelling"/>
   <addaction name="Process"/>
   <addaction name="performance_evaluation"/>
   <addaction name="menuSAR"/>
   <addaction name="menuInSAR"/>
   <addaction name="menuDInSAR"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/new_project.png</normaloff>:/QtWidgetsApplication3/bin/icon/new_project.png</iconset>
   </property>
   <property name="text">
    <string>新建工程</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/open_project.png</normaloff>:/QtWidgetsApplication3/bin/icon/open_project.png</iconset>
   </property>
   <property name="text">
    <string>打开工程</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/recen_open.png</normaloff>:/QtWidgetsApplication3/bin/icon/recen_open.png</iconset>
   </property>
   <property name="text">
    <string>最近打开</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/save.png</normaloff>:/QtWidgetsApplication3/bin/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>保存工程</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/saveas.png</normaloff>:/QtWidgetsApplication3/bin/icon/saveas.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/saveall.png</normaloff>:/QtWidgetsApplication3/bin/icon/saveall.png</iconset>
   </property>
   <property name="text">
    <string>全部保存</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normaloff>../bin/icon/close.png</normaloff>../bin/icon/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/quit.png</normaloff>:/QtWidgetsApplication3/bin/icon/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionRegistration">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/coregistration.png</normaloff>:/QtWidgetsApplication3/bin/icon/coregistration.png</iconset>
   </property>
   <property name="text">
    <string>配准</string>
   </property>
  </action>
  <action name="actionInterferometric_Formation">
   <property name="text">
    <string>Interferometric Formation</string>
   </property>
  </action>
  <action name="actionDenoise">
   <property name="text">
    <string>Denoise</string>
   </property>
  </action>
  <action name="actionUnwrap">
   <property name="text">
    <string>Unwrap</string>
   </property>
  </action>
  <action name="actionDEM">
   <property name="text">
    <string>DEM</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/cut.png</normaloff>:/QtWidgetsApplication3/bin/icon/cut.png</iconset>
   </property>
   <property name="text">
    <string>裁剪</string>
   </property>
  </action>
  <action name="actionRepeatPass">
   <property name="text">
    <string>重轨模式</string>
   </property>
  </action>
  <action name="actionSingleTransDoubleRecv">
   <property name="text">
    <string>一发双收模式</string>
   </property>
  </action>
  <action name="actionBaseline_Preview">
   <property name="text">
    <string>Baseline Preview</string>
   </property>
  </action>
  <action name="actionS1_TOPS_BackGeocoding">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/coregistration.png</normaloff>:/QtWidgetsApplication3/bin/icon/coregistration.png</iconset>
   </property>
   <property name="text">
    <string>S1 TOPS BackGeocoding</string>
   </property>
  </action>
  <action name="actionS1_Deburst">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/splice.png</normaloff>:/QtWidgetsApplication3/bin/icon/splice.png</iconset>
   </property>
   <property name="text">
    <string>S1 Deburst</string>
   </property>
  </action>
  <action name="actionSLC_deramp">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/dem.png</normaloff>:/QtWidgetsApplication3/bin/icon/dem.png</iconset>
   </property>
   <property name="text">
    <string>SLC_deramp</string>
   </property>
  </action>
  <action name="actionBaseline_Formation">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/baseline_formation.png</normaloff>:/QtWidgetsApplication3/bin/icon/baseline_formation.png</iconset>
   </property>
   <property name="text">
    <string>Baseline Formation</string>
   </property>
  </action>
  <action name="actionSBAS_deformation">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/time_series.png</normaloff>:/QtWidgetsApplication3/bin/icon/time_series.png</iconset>
   </property>
   <property name="text">
    <string>Time-Series Analysis</string>
   </property>
  </action>
  <action name="actionDeformation_Preview">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/view.png</normaloff>:/QtWidgetsApplication3/bin/icon/view.png</iconset>
   </property>
   <property name="text">
    <string>Time Series View</string>
   </property>
  </action>
  <action name="actionreference_re_selection">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/reference.png</normaloff>:/QtWidgetsApplication3/bin/icon/reference.png</iconset>
   </property>
   <property name="text">
    <string>reference re-selection</string>
   </property>
  </action>
  <action name="actionExport_KML">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/GoogleEarth.png</normaloff>:/QtWidgetsApplication3/bin/icon/GoogleEarth.png</iconset>
   </property>
   <property name="text">
    <string>export as kml</string>
   </property>
  </action>
  <action name="actiongeocode">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/geocoding.png</normaloff>:/QtWidgetsApplication3/bin/icon/geocoding.png</iconset>
   </property>
   <property name="text">
    <string>地理编码</string>
   </property>
  </action>
  <action name="actionS1_frame_merge">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/frame_merge.png</normaloff>:/QtWidgetsApplication3/bin/icon/frame_merge.png</iconset>
   </property>
   <property name="text">
    <string>S1 frame merge</string>
   </property>
  </action>
  <action name="actionS1_swath_merge">
   <property name="icon">
    <iconset resource="../QtWidgetsApplication3.qrc">
     <normaloff>:/QtWidgetsApplication3/bin/icon/swath_merge.png</normaloff>:/QtWidgetsApplication3/bin/icon/swath_merge.png</iconset>
   </property>
   <property name="text">
    <string>S1 swath merge</string>
   </property>
  </action>
  <action name="actionPingPong">
   <property name="text">
    <string>乒乓模式</string>
   </property>
  </action>
  <action name="actionDualFreqPingPong">
   <property name="text">
    <string>双频乒乓模式</string>
   </property>
  </action>
  <action name="actionlvbo1">
   <property name="text">
    <string>普通滤波</string>
   </property>
  </action>
  <action name="actionlvbo2">
   <property name="text">
    <string>双频乒乓模式滤波</string>
   </property>
  </action>
  <action name="actionjiechan1">
   <property name="text">
    <string>普通解缠</string>
   </property>
  </action>
  <action name="actionjiechan2">
   <property name="text">
    <string>双频乒乓解缠</string>
   </property>
  </action>
  <action name="actiongaocheng">
   <property name="text">
    <string>高程反演</string>
   </property>
  </action>
  <action name="actioncoregis_evaluate">
   <property name="text">
    <string>配准评估</string>
   </property>
  </action>
  <action name="actionunwrap_evaluate">
   <property name="text">
    <string>解缠评估</string>
   </property>
  </action>
  <action name="actiondem_evaluate">
   <property name="text">
    <string>高程评估</string>
   </property>
  </action>
  <action name="actionqucankao">
   <property name="text">
    <string>去除参考相位</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeView</class>
   <extends>QTreeView</extends>
   <header>treeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../QtWidgetsApplication3.qrc"/>
 </resources>
 <connections/>
</ui>
